docs/gettingstarted/index.rst

```diff
@@ -132,12 +129,8 @@ Getting started with Flocker
 
          Install the Flocker client on your Linux machine:
 
-         .. version-code-block:: console
-
-            you@laptop:~$ sudo apt-get update && sudo apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \
-              virtualenv flocker-tutorial && \
-              flocker-tutorial/bin/pip install --upgrade pip && \
-              flocker-tutorial/bin/pip install --quiet https://clusterhq-archive.s3.amazonaws.com/python/Flocker-|latest-installable|-py2-none-any.whl && source flocker-tutorial/bin/activate
+         .. task:: install_cli ubuntu-14.04
```

Comment:
Yes, the `&&` command lines are obnoxious. It spectacularly obfuscates the commands and fails to achieve the primary goal of informing the user of the steps required to get going, pretty much the goal of a Getting Started Guide.  Not to mention, when they do get an error, they will have little idea which part of the command failed.